### ISSUE TITLE
Broken support for ^2.0.9

### ISSUE THREAD

**Issue by BRA1L0R** [kriasoft/hyperapp-render#22]
**I'm submitting a ...**

* [x] bug report
* [ ] feature request
* [ ] other (Please do not submit support requests here (below))

### Description of the bug

hyperapp-render will not work with anything newer than 2.0.9 (included). This is because the creator of hyperapp made a commit where he switched the name of two keys in the vdom object, see commit [8e6a4908954186ed8466ae9ef15ec172ddbea2d0](https://github.com/jorgebucaran/hyperapp/commit/8e6a4908954186ed8466ae9ef15ec172ddbea2d0).

### Steps to reproduce

```js
import { renderToString } from 'hyperapp-render'
import { h, text } from 'hyperapp'

console.log(renderToString(h("p", {}, text("Hello World!")))) // output: undefined
```

With version 2.0.9 the output is: `undefined`

### Expected output

With version 2.0.8 the output of the snippet in the section above is: `<p>Hello World!</p>`

**Comment by frenzzy** [comment]
Thank you for the bug report! Just released [v4.0.0](https://github.com/kriasoft/hyperapp-render/releases/tag/v4.0.0) :tada: with support for Hyperapp v2.0.9. Demo: [CodePen](https://codepen.io/frenzzy/pen/BaoGbXK/left/?editors=0010)

**Comment by BRA1L0R** [comment]
Wow how fast
